docs/docs/mapping/customizing_your_gateway.md

```diff
@@ -358,3 +358,32 @@ HTTP statuses and their mappings to gRPC statuses:
 - HTTP `400 Bad Request` -> gRPC `3 INVALID_ARGUMENT`
 
 This method is not used outside of the initial routing.
+
+### Customizing Routing Errors
+
+If you want to retain HTTP `405 Method Not Allowed` instead of allowing it to be converted to the equivalent of the gRPC `12 UNIMPLEMENTED`, which is  HTTP `501 Not Implmented` you can use the following example:
+
+```go
+func handleRoutingError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w http.ResponseWriter, r *http.Request, httpStatus int) {
+	if httpStatus != http.StatusMethodNotAllowed {
+		runtime.DefaultRoutingErrorHandler(ctx, mux, marshaler, writer, request, httpStatus)
+
```

Comment:
Rogue newline 😂.

Reply:
Linter habits, https://github.com/ssgreg/nlreturn is enabled for local projects and it's become part of my muscle memory now... ;-)